spec/features/course/assessment/answer/text_response_answer_spec.rb

```diff
@@ -40,6 +40,16 @@
 
       scenario 'I can view the grading scheme' do
         visit edit_course_assessment_submission_path(course, assessment, submission)
+        click_link I18n.t('course.assessment.submission.submissions.worksheet.auto_grade')
+        wait_for_job
+
+        if submission.answers.first.correct?
```

Comment:
Yes, the specs should still count towards coverage. In any case, I think it's not meaningful to have a branch here because you should know the result of the grading.